src/inhouse.js

```diff
@@ -0,0 +1,250 @@
+const https = require('https');
+const config = require('./config.json');
+const riotKey = config.riotKey;
```

Comment:
can simplify last two lines to just `const riotKey = require('./config.json).riotKey`
since we don't really use `config` outside of this.